scripts/write-config.sh

```diff
@@ -322,6 +322,8 @@ landingzone:
       - leonardo-dev@broad-dsde-dev.iam.gserviceaccount.com
       - workspace-wsmtest@terra-kernel-k8s.iam.gserviceaccount.com
       - Elizabeth.Shadowmoon@test.firecloud.org
+pactbroker:
```

Comment:
This doesn't actually work to provide a local override -- the pactbroker.url defined in `application-test.yml` ends up taking precedence.

Reply:
if this doesn't work, then can you delete it? If it needs to be present for some reason, add a code comment explaining it doesn't have any effect.